.gitignore

```diff
@@ -1,5 +1,5 @@
 # Project-specific files
```

Comment:
Can we remove the `siftnlp` folder from root if there isn't anything in it? Maybe replace with a `lib` folder for functions common to multiple jobs?